modules/virt-importing-vm-cli.adoc

```diff
@@ -169,7 +169,7 @@ EOF
 <9> If `storageMappings` are specified in both the ResourceMapping and the VirtualMachineImport CRs, the VirtualMachineImport CR takes precedence.
 <10> Specify the source storage domain.
 <11> Specify the target storage class.
-<12> Specify the source VM disk UUID, for example, `8181ecc1-5db8-4193-9c92-3ddab3be7b05`.
+<12> Specify the source VM disk ID, for example, `8181ecc1-5db8-4193-9c92-3ddab3be7b05`. You can obtain the disk ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine to display the details of a VM.
```

Comment:
This second sentence sounds like it's running on. I would think that it should end at "You can obtain the disk ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine." Or that it needs and "and" in there before the rest of the sentence.

Same with the similar sentence for callout 3.

Reply:
@bergerhoffer 
I'm not sure that splitting the sentence makes sense. Running the API call displays the details of a VM. 

I added "to display the details of a VM" so that the user does not think that the command shows only the disk ID. I could remove that clause if you think it's unnecessary but I would not split the sentence there or add "and" ("and displaying the details of a VM").

Or am I not understanding where the "and" would go? Please clarify.

Reply:
@apinnick So I assume the intent is how to get that ID. If I understand correctly, first I assume that I need to open a web browser and go to that page, and then look at the details of the VM to find the ID. The way that the sentence is worded right now just doesn't seem right with "by entering" and then "to display". 

Would something like this mean the same thing?

```
To obtain the disk ID, enter `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine and review the details of the VM.
```

Or:

```
You can obtain the disk ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine and reviewing the details of the VM to find the disk ID.
```

Or, two sentences:

```
To obtain the disk ID, enter `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine. This page displays the details of the VM, including the disk ID.
```

Or, maybe you could work in something that you can "find" the ID on that page, so they know they have to look for it.

I know I'm not being too helpful/decisive here :) It's just worth another look.

Reply:
OK, now I understand what you're saying. I'll update the section.

Reply:
Thanks, I think that is better. Merging!